### PR TITLE
Loginout

### PR DESCRIPTION
**Pull request by 2mnyzs**
Login Logout funcitonality
- MySQL script to create needed tables
- Page for logging in and requesting a password reset
- Page for resetting password

### CODE REVIEW

**Comment by MatthewMeacham** [comment]
Suppress Exceptions in PROD configruation

File: register/index.php [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                            <button type='submit' class='btn btn-primary pull-right'>Submit</button>
+                        </section>
+                    </form>
+                </div>
+                </div>
+                </div>
+            </div>
+        </div>
+    <?php
+            }else{
```
Comment (JakeKo) [inline register/index.php]: Indenting here makes it a bit difficult to read/understand

File: mysql scripts/loginDB.sql [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+CREATE TABLE `PasswordReset` (
+	`userId` INT UNSIGNED NOT NULL,
+	`token` VARCHAR(255) NOT NULL,
+	`ipAddress` VARCHAR(95) NOT NULL,
+	`timestamp` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+	`archived` TINYINT NOT NULL DEFAULT 0,
+	foreign key (userId) references User(userId)
+	);
+
+INSERT INTO `User`
```
Comment (JakeKo) [inline mysql scripts/loginDB.sql]: When the style guide for SQL is finalized, this will be need spacing changes

File: login/index.php [hunk 3]
```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html class="no-js" lang="">
+<?php
+    require_once '../server/header.php';
+?>
+    <body>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```
Comment (JakeKo) [inline login/index.php]: Are we planning on using bootstrap until I can get to styling?

**Pull request merged**